workflow/rules/mapping.smk

```diff
@@ -37,7 +37,7 @@ rule annotate_umis:
     params:
         extra=get_umi_read_structure,
     resources:
-        mem_gb="10",
+        mem_mb=10000,
```

Comment:
```suggestion
        mem="10GB",
```

Reply:
Are we sure about this? Is it really a constant?

Reply:
I would also go for some dynamic solution. I just had a 21gb bam-file where fgbio returned a memory error with 10GB set.